contribution/extension/type.php

```diff
@@ -139,6 +139,14 @@ public function perform_repack(\titania_contribution $contrib, \titania_revision
 		try
 		{
 			$this->repack($package, $contrib, $revision);
+			$repack_complete = $this->user->lang('NEW_REVISION_REPACK_COMPLETE');
+
+			// Oversized packages are over 2MB
+			if ($package->get_size() > 2097152)
+			{
+				$repack_complete .= $this->user->lang('NEW_REVISION_REPACK_OVERSIZE');
```

Comment:
There a way to only append this if the queue is enabled for the contribution type?

Reply:
How do you mean? Should it get to the point of repacking if the queue is not enabled?

Reply:
I might be mistaken, but I think this process runs independently from if the queue is enabled or not. If that's not the case then you can just ignore the comment :)

Reply:
How do you disable an individual queue?

Reply:
Nevermind this. I'm thinking the forum topic that gets all messages as well. This is fine as is.